homeassistant/components/environment_canada/camera.py

```diff
@@ -40,7 +40,11 @@
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Set up the Environment Canada camera."""
-    from env_canada import ECRadar
+    try:
+        from env_canada import ECRadar
+    except ImportError:
+        raise ImportError('Missing prerequsites. Run: sudo apt-get install '
```

Comment:
The user is now no longer informed what is actually wrong: their numpy installation is borked. Please add that to the message or else it is impossible to diagnose if you're not on a debian based system.

Also, don't raise an ImportError, just log an error and return.

Reply:
I think that it's better to just point people to a site with info than trying to put all solutions in the error message. Imagine if we need to get the solutions for every possible LInux installation in this error message.